src/src/WebServer/Metrics.cpp

```diff
@@ -61,130 +61,115 @@
 
 
 void handle_metrics() {
+    TXBuffer.startStream();
+
+
     int result;
     String resultString;
     resultString.reserve(1000);
 
 
     //uptime
-    resultString = F("# HELP espeasy_uptime current device uptime in minutes\n");
-    resultString += F("# TYPE espeasy_uptime counter\n");        
-    resultString += "espeasy_uptime ";
-    resultString += getValue(LabelType::UPTIME);       
-    resultString += "\n";
-  
+    addHtml(F("# HELP espeasy_uptime current device uptime in minutes\n"));
+    addHtml(F("# TYPE espeasy_uptime counter\n"));
+    addHtml("espeasy_uptime ");
+    addHtml(getValue(LabelType::UPTIME));       
+    addHtml("\n");    
 
     //load
-    resultString += F("# HELP espeasy_load device percentage load\n");
-    resultString += F("# TYPE espeasy_load gauge\n");         
-    resultString += "espeasy_load ";
-    resultString += getValue(LabelType::LOAD_PCT);     
-    resultString += "\n";
+    addHtml(F("# HELP espeasy_load device percentage load\n"));
+    addHtml(F("# TYPE espeasy_load gauge\n"));
+    addHtml("espeasy_load ");
+    addHtml(getValue(LabelType::LOAD_PCT));
+    addHtml("\n");
 
     //Free RAM
-    resultString += F("# HELP espeasy_free_ram device amount of RAM free in Bytes\n");
-    resultString += F("# TYPE espeasy_free_ram gauge\n");    
-    resultString += "espeasy_free_ram ";
-    resultString += getValue(LabelType::FREE_MEM);   
-    resultString += "\n";
+    addHtml(F("# HELP espeasy_free_ram device amount of RAM free in Bytes\n"));
+    addHtml(F("# TYPE espeasy_free_ram gauge\n"));
+    addHtml("espeasy_free_ram ");
+    addHtml(getValue(LabelType::FREE_MEM));
+    addHtml("\n");
```

Comment:
`addHtml('\n');`
`\n` is a single char, and way less computational expensive compared to a string

Reply:
Ha, can you tell I've never written any C/C++ in anger? :D

Reply:
"Anger"?? relax :)